docs/admin/federation/kubefed.md

```diff
@@ -0,0 +1,172 @@
+---
+assignees:
+- madhusudancs
+
+---
+Kubernetes v1.5 introduced a new command line tool called `kubefed`
+to facilitate the administration of cluster federations. `kubefed`
+stands for "Kubernetes Federate". It aids in both deploying a
+Kubernetes Cluster Federation control plane and adding/removing
```

Comment:
"`kubefed` helps you to deploy a new Kubernetes cluster federation control plane, and to add clusters to or remove clusters from an existing federation control plane."